packages/bff/src/routers/genericRouter.ts

```diff
@@ -6,17 +6,44 @@ import {
   zodiosValidationErrorToApiProblem,
 } from "pagopa-interop-commons";
 import { bffApi } from "pagopa-interop-api-clients";
+import { toolsServiceBuilder } from "../services/toolsService.js";
+import { PagoPAInteropBeClients } from "../providers/clientProvider.js";
+import { fromBffAppContext } from "../utilities/context.js";
+import { emptyErrorMapper } from "../utilities/errorMappers.js";
+import { makeApiProblem } from "../model/domain/errors.js";
 
 const toolsRouter = (
```

Comment:
🤔  I think the base branch is very outdated with respect to `main` - we don't have a `genericRouter.ts` file anymore, @sandrotaje split it into two dedicated files `supportRouter.ts` and `toolRouter.ts` - maybe you could do the same in this PR and when `main` is merged you will just get the same changes from there

Reply:
For now i manually ported this modification in order to have less conflicts